src/qt/forms/debugwindow.ui

```diff
@@ -6,13 +6,16 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>740</width>
-    <height>430</height>
+    <width>744</width>
+    <height>531</height>
```

Comment:
Why this change?

Reply:
I am not sure of the reason. But I observed they are set to these values (744, 531) each time debugwindow.ui is opened in QTCreator even if we try to change them. However, this does not cause any functional change.

Reply:
Ok, then seems reasonable to change to these values.

Reply:
We manually edit these values to remove this diff junk - this way it stays consistent.

Reply:
Including - resetting the currentIndex to zero (below)

Reply:
Resetted these changes to the master's value in 0c364201a2e16355cd6424c6137761c7967db4e0